app/helpers/application_helper.rb

```diff
@@ -107,7 +107,7 @@ def error_message_alert
   end
 
   def onts_for_select
```

Comment:
here change it to have include_views disabled b y default, and enable it only for the annotator, otherwise as this helper is used all over the place we can have unexpected behaviors/bugs.
do soothing like this: 
```ruby
  def onts_for_select(include_views: false)
    ontologies = LinkedData::Client::Models::Ontology.all({include: "acronym,name", include_views: include_views})
    ...
 end
```

Reply:
done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/784/commits/b05690131c8e194dff46ddf1eca6597d5814261d